spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -16,8 +16,16 @@ local CLUSTERING_SYNC_STATUS = require("kong.constants").CLUSTERING_SYNC_STATUS
 local KEY_AUTH_PLUGIN
 
 
+local confs = {
+  wrpc = "spec/fixtures/custom_nginx.template",
+  json = "spec/fixtures/json_nginx.template",
+}
+-- disable wrpc in CP
+os.execute(string.format("cat %s | sed 's/wrpc/json/g' > %s", confs.wrpc, confs.json))
```

Comment:
```suggestion
os.execute(string.format("cat %s | sed 's/wrpc/foobar/g' > %s", confs.wrpc, confs.json))
```

is probably better? It is not `json` after all and this name might be confusing.